common/network_constants.cc

```diff
@@ -38,6 +42,9 @@ const char kCRLSetPrefix3[] =
 const char kCRLSetPrefix4[] =
     "*://storage.googleapis.com/update-delta/hfnkpimlhhgieaddgfemjhofmfblmnib"
     "/*crxd";
+const char kGoogleAPIPrefix[] = "https://www.googleapis.com/";
```

Comment:
out of curiosity, why `https` instead of `*` here?

Reply:
it was not connecting over `http` but I agree, using * would help if the code fallsback to the non-secure url